docs/contributing/gatsby-governance-model.md

```diff
@@ -35,28 +35,28 @@ If you’d like to request or propose a Gatsby feature (or a change to an existi
 
 ### Community PR and Issue Review Process
 
-Issues submitted by the community go through our Triage Process. You can learn more about this process in our [Triaging GitHub Issues doc](/docs/contributing/triaging-github-issues/).
+Issues submitted by the community go through our Triage Process. You can learn more about this process in our [Triaging GitHub Issues doc](/contributing/triaging-github-issues/).
 
-Our pull request review process is outlined in our [Managing Pull Requests doc](/docs/contributing/managing-pull-requests/).
+Our pull request review process is outlined in our [Managing Pull Requests doc](/contributing/managing-pull-requests/).
 
 If you have any questions about an issue or PR, you can add a comment to the issue/PR on GitHub or reach out to us via one of the options in the "Engage with the Gatsby Team" section below.
 
 ## Code of Conduct
 
 In the interest of fostering an open and welcoming environment, we as contributors and maintainers pledge to making participation in our project and our community a harassment-free experience for everyone, regardless of age, body size, disability, ethnicity, gender identity and expression, level of experience, nationality, personal appearance, race, religion, or sexual identity and orientation.
 
-[Read the full code of conduct](/docs/contributing/code-of-conduct/)
+[Read the full code of conduct](/contributing/code-of-conduct/)
 
 ## Guidance and documentation for contributors
 
 Here are some relevant docs and resources for contributors:
 
-- [Contributing to Gatsby.js](/docs/contributing)
```

Comment:
These links are fine, `/docs/` is required since the files are within the `docs` directory. What is missing is `.md` at the end of links which is why you are getting 404.

E.g.
[https://github.com/gatsbyjs/gatsby/blob/master/docs/contributing/how-to-contribute]() gives an error [https://github.com/gatsbyjs/gatsby/blob/master/docs/contributing/how-to-contribute.md]() does not.

Reply:
no - these docs are intendet to read at gatsbyjs.org and not at github.com

check:

https://gatsbyjs.org/contributing/how-to-contribute -> ok
https://gatsbyjs.org/docs/contributing/how-to-contribute -> 404

Reply:
My apologies, I tested it works locally.